taskcluster/ci/config.yml

```diff
@@ -20,11 +20,21 @@ workers:
       implementation: docker-worker
       os: linux
       worker-type: 'b-linux'
+    b-linux-gcp:
+      provisioner: 'app-services-{level}'
+      implementation: docker-worker
+      os: linux
+      worker-type: 'b-linux-gcp'
```

Comment:
What if instead of adding a second section here, we just modify the `worker-type` key of the existing `b-linux` section? So e.g:
```
b-linux:
    ...
    worker-type: b-linux-gcp
```

This way:
A) We won't have to clean up the non-GCP pools, and
B) It'll mean we don't have to update any of the task definitions (also we won't accidentally miss updating some of them)

I think duplicating these for GCP made sense in Gecko where there were a _ton_ of tasks, but for most projects on Github it's probably not necessary as we can just do all tasks at once.